server/src/test/java/io/crate/expression/scalar/ConcatwsFunctionTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar;
+
+import org.junit.Test;
+import static io.crate.testing.SymbolMatchers.isLiteral;
+
+public class ConcatwsFunctionTest extends ScalarTestCase {
+
+    @Test
+    public void testMinimumArguments() {
+        assertNormalize("concat_ws(',','foo')",
+            isLiteral("foo"));
+    }
+
+    @Test
+    public void testTwoStringArguments() {
+        assertNormalize("concat_ws(',','foo','bar')",
+            isLiteral("foo,bar"));
+    }
+
+    @Test
+    public void test4Arguments(){
+        assertNormalize("concat_ws(',','foo','bar','joe')",
+            isLiteral("foo,bar,joe"));
+    }
```

Comment:
assuming that we would remove special handling of the last element, `testManyStrings` should be fine and these 3 cases can be dropped.